lib/Agrammon/OutputFormatter/PDF.pm6

```diff
@@ -162,6 +164,17 @@ sub input-output-as-pdf(
         $include-filters, $all-filters,
     );
 
+    my @log;
+    %data<log> = @log;
+    my @entries := $outputs.log-collector.entries;
+    for @entries -> $_ {
+        @log.push(%(
+            message  => latex-escape(.messages{$language}),
+            gui      => .gui{$language},
+            instance => latex-escape(.instance),
+        ));
+    }
+
```

Comment:
I see `@log` being set up, but then not used anywhere

Reply:
> I see `@log` being set up, but then not used anywhere

It is used on the template:
```
<@log>
...
</@>
```

Reply:
Oh, I totally missed the line `%data<log> = @log;` - probably because I tend to set things up and *then* stash them where they are going, so expected it to be at the end. :-)